File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int> hashSet(nums.begin(), nums.end());
        int max = 0;
        for(const int& num : nums)
        {
            int curr = num;
            int seq = 1;
            if(hashSet.find(num - 1) != hashSet.end())
                continue;
            
            while(hashSet.find(++curr)!=hashSet.end())
                seq++;
            if(seq > max)
                max = seq;
        }
        return max;
        
    }
};

File: Data Structures & Algorithms/duplicate-integer/submission-1.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        if(len(nums)==len(set(nums))):
            return False
        return True

        # for i in range(len(nums)-1):
        #     if nums[i] in nums[i+1:]:
        #         return True
        
        # return False
        